webapp/src/components/wrappers/MaintenanceWrapper/index.tsx

```diff
@@ -47,88 +47,106 @@ function MaintenanceWrapper(props: Props) {
   const maintenanceMode = useAppSelector(getMaintenanceMode);
   const dispatch = useAppDispatch();
 
-  const onClick = () => {
+  const res = usePromiseWithSnackbarError(
```

Comment:
You removed necessary code, keep the old `useEffect`. 
`usePromiseWithSnackbarError` is not needed here.